### PR TITLE
docker: added kibana and DB name

### PR DESCRIPTION
**Pull request by krzysztof**
Signed-off-by: Krzysztof Nowak <k.nowak@cern.ch>

### CODE REVIEW

File: {{cookiecutter.project_shortname}}/docker-services.yml [hunk 1]
```diff
@@ -66,6 +67,12 @@ services:
     ports:
       - "9200:9200"
       - "9300:9300"
+  kibana:
+    image: docker.elastic.co/kibana/kibana:6.2.2
```
Comment (slint) [inline {{cookiecutter.project_shortname}}/docker-services.yml]: This should be the `docker.elastic.co/kibana/kibana-oss:6.2.3` flavor: https://www.elastic.co/guide/en/kibana/current/_pulling_the_image.html

**Comment by lnielsen** [comment]
Why are we adding Kibana?

**Comment by slint** [comment]
The main reason is probably using [the `Sense` plugin](https://www.elastic.co/blog/found-sense-a-cool-json-aware-interface-to-elasticsearch) (it used to be a plugin, now it's part of Kibana), for testing queries easily.

File: {{cookiecutter.project_shortname}}/docker-services.yml [hunk 2]
```diff
@@ -72,7 +73,7 @@ services:
     {% if cookiecutter.elasticsearch == 'elasticsearch6'%}
     image: docker.elastic.co/elasticsearch/elasticsearch-oss:6.2.2
     {% elif cookiecutter.elasticsearch == 'elasticsearch5' %}
-    image: elasticsearch:5.6
+    image: elasticsearch:5.6.4
```
Comment (diegodelemos) [inline {{cookiecutter.project_shortname}}/docker-services.yml]: @slint why did we choose DockerHub for ES 5?

**Pull request merged**